### ISSUE TITLE
Cannot restore Individual Wallet

### ISSUE THREAD

**Issue by Charlie953A** [ontio/OWallet#81]
I have created an individual wallet and ledger wallet in Owallet, to perform token swap from NEO-wallet to ledger. I swapped 10 ONT as a test before sending my entire amount. However, now when I open Owallet the individual wallet is gone. I have all the backups: keystore file, private key, mnemonic phrase, pass word, etc. But whatever I try, I cannot import the individual wallet (I hit 'import', there's some 'thinking' and then nothing, no errors either). Does anyone have any suggestions I can try?
I can access the ledger wallet I already created, so the app is working.... I'm really lost here.

**Comment by MickWang** [comment]
Hi, which version do you use and what is your operating system?
Have you set the path to save the keystore.db at the very first beginning?Or have you change the path? Or have you delete the keystore.db file?

**Comment by Charlie953A** [comment]
Yesterday I eventually managed to get access by migrating the app back to my old laptop. I used the individual wallet only for the swap, and after that I have transferred my ONT and ONG to a Ledger Owallet. That one I can also open on my new laptop. 

I still had everything, including the keystore file, but was not able to import the individual wallet on my new laptop. So I managed a workaround and will now leave the empty individual wallet and only use the ledger wallet.

Thanks for trying to help. I will now close this issue.
